PetProject/Screens/GamesList/ViewModel+VC/GamesListViewModel.swift

```diff
@@ -13,44 +13,55 @@ class GamesListViewModel: ObservableObject {
     @Published private(set) var store: StoreModel
     @Published private(set) var dealsList: [DealModel] = []
     @Published private(set) var dealsImages: [DealModel.ID : UIImage] = [:]
+    @Published private(set) var errorMessage: String?
     private(set) var dealsListIsFull = false
     
     private let apiService: CheapSharkService
     private var paginationPage = 1
     private let paginationQueue = DispatchQueue(label: "GamesListPaginationQueue")
     private let paginationDispatchGroup = DispatchGroup()
     private var cancellables = Set<AnyCancellable>()
+    private var searchGameTitle = ""
     
     init(store: StoreModel, apiService: CheapSharkService) {
         self.store = store
         self.apiService = apiService
-        paginateDealsList()
+        DispatchQueue.main.async { [weak self] in
```

Comment:
I think you are creating a GameListViewModel in the main queue, or am I wrong?

Reply:
Done